app/core/Tracker.coffee

```diff
@@ -137,6 +143,9 @@ module.exports = class Tracker extends CocoClass
       gaFieldObject.eventLabel = properties.label if properties.label?
       gaFieldObject.eventValue = properties.value if properties.value?
 
+      gaFieldObject.dimmension0 = me.getHomePageTestGroup()
```

Comment:
Do we still need this?

Reply:
If this test has concluded I'll remove it from the dimensions we save.

Reply:
This test is no longer required and `getHomePageTestGroup` returns undefined.

Reply:
That `me.getHomePageTestGroup` has ended according to the last commit. @shubhi1092, is it safe to remove?
And tiny nitpick of extra `m` in dimension.

Reply:
Yes, it is safe to remove

Reply:
Thanks for catching the typo that is critical for GA reporting.  I've removed the old homepage test.